### PR TITLE
Gateway spoilers, week 1

### PR DESCRIPTION
**Pull request by NoahTheDuke**

### CODE REVIEW

**Review by plural — CHANGES_REQUESTED** [review]
you're missing some required fields and the build is failing for this.

File: pack/sg.json [hunk 1]
```diff
@@ -1 +1,93 @@
...
+        "code": "30001",
+        "faction_code": "anarch",
+        "influence_limit": 15,
+        "keywords": "G-mod",
+        "minimum_deck_size": 40,
+        "pack_code": "sg",
+        "position": 1,
+        "quantity": 1,
+        "side_code": "runner",
+        "text": "The first time each turn you trash a card you are accessing, you may draw 1 card and gain 1[c].",
```
Comment (gereons) [inline pack/sg.json:13]: [c] should be [credit]

File: pack/sg.json [hunk 2]
```diff
@@ -1 +1,93 @@
...
+        "cost": 2,
+        "deck_limit": 3,
+        "faction_code": "shaper",
+        "faction_cost": 3,
+        "keywords": "Console",
+        "pack_code": "sg",
+        "position": 23,
+        "quantity": 3,
+        "side_code": "runner",
+        "text": "+1[mu]\nWhenever an agenda is scored or stolen, you may gain 1[c]. Then, you may install 1 card.\nLimit 1 console per player.",
```
[the remark this answers is not in the document]
Comment (gereons) [reply pack/sg.json:44]: Also, [c] should be [credit]

**Review by gereons — CHANGES_REQUESTED** [review]

**Review by plural — APPROVED** [review]
thanks, Noah!

**Review by gereons — APPROVED** [review]

**Pull request merged**